Runnect-iOS/Runnect-iOS/Presentation/MyPage/VC/InfoVC/GoalRewardInfoVC.swift

```diff
@@ -6,18 +6,187 @@
+        GoalRewardInfoModel(stampImg: ImageLiterals.imgStampR1, stampStandard: "달리기 유망주"),
+        GoalRewardInfoModel(stampImg: ImageLiterals.imgStampR2, stampStandard: "아마추어 선수"),
+        GoalRewardInfoModel(stampImg: ImageLiterals.imgStampR2, stampStandard: "마라톤 선수")
+    ]
+    
+    // MARK: - Constants
+    
+    final let stampInset: UIEdgeInsets = UIEdgeInsets(top: 32, left: 34, bottom: 10, right: 34)
+    final let stampLineSpacing: CGFloat = 20
+    final let stampItemSpacing: CGFloat = 26
+    final let stampCellHeight: CGFloat = 112
+    
+    // MARK: - View Life Cycle
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        setNavigationBar()
         setUI()
-        // Do any additional setup after loading the view.
+        setLayout()
+        register()
     }
+    
 }
 
+// MARK: - Methods
+
+// MARK: - @objc Function
+
+// MARK: - UI & Layout
+
 extension GoalRewardInfoVC {
+    
+    // MARK: - Layout Helpers
+    
+    private func setNavigationBar() {
+        view.addSubview(navibar)
+        
+        navibar.snp.makeConstraints { make in
+            make.top.leading.trailing.equalTo(view.safeAreaLayoutGuide)
+            make.height.equalTo(48)
+        }
+    }
+    
     private func setUI() {
         view.backgroundColor = .w1
+        stampBottomView.backgroundColor = .m3
+        stampCollectionView.backgroundColor = .m3
     }
+    
+    private func setLayout() {
+        view.addSubviews(stampTopView, stampBottomView)
+        
+        stampTopView.snp.makeConstraints { make in
+            make.top.equalTo(navibar.snp.bottom)
+            make.leading.trailing.equalTo(view.safeAreaLayoutGuide)
+            make.height.equalTo(235)
+        }
+        
+        stampTopView.addSubviews(stampImage, stampExcourageLabel)
+        
+        stampImage.snp.makeConstraints { make in
+            make.top.equalToSuperview()
+            make.centerX.equalToSuperview()
+            make.width.equalTo(181)
+            make.height.equalTo(167)
+        }
+        
+        stampExcourageLabel.snp.makeConstraints { make in
+            make.top.equalTo(stampImage.snp.bottom).offset(16)
+            make.centerX.equalToSuperview()
+        }
+        
+        stampBottomView.snp.makeConstraints { make in
+            make.top.equalTo(stampTopView.snp.bottom)
+            make.leading.trailing.equalTo(view.safeAreaLayoutGuide)
+            make.height.equalTo(view.safeAreaLayoutGuide).inset(80)
+        }
+        
+        stampBottomView.addSubview(stampCollectionView)
+        
+        stampCollectionView.snp.makeConstraints { make in
+            make.top.leading.trailing.equalToSuperview()
+            make.height.equalTo(calculateCellHeight())
+        }
+    }
+    
+    // MARK: - General Helpers
+
+    private func register() {
+        stampCollectionView.register(GoalRewardInfoCVC.self,
+                                     forCellWithReuseIdentifier: GoalRewardInfoCVC.identifier)
```

Comment:
```suggestion
                                     forCellWithReuseIdentifier: GoalRewardInfoCVC.className)
```
으로 사용하면 identifier를 만들어줄 필요가 없어요